src/build.cc

```diff
@@ -97,7 +97,7 @@ void BuildStatus::BuildEdgeStarted(Edge* edge) {
   ++started_edges_;
 
   if (edge->use_console() || printer_.is_smart_terminal())
-    PrintStatus(edge);
+    PrintStatus(edge, false);
```

Comment:
I always find bare bool parameters like this mysterious. What does `false` mean here? Either write `PrintStatus(edge, /*edge_finished=*/false);` here, or add `enum EdgeStatus { kEdgeStarted, kEdgeFinished };` above the PrintStatus() prototype and write `PrintStatus(edge, kEdgeStarted);` here.